test/stripe/invoice_test.rb

```diff
@@ -73,6 +73,26 @@ class InvoiceTest < Test::Unit::TestCase
         assert invoice.kind_of?(Stripe::Invoice)
       end
 
+      should "retrieve upcoming invoices with items" do
+        items = [{
+          plan: 'gold',
+          quantity: 2
+        }]
+
+        invoice = Stripe::Invoice.upcoming(
+          customer: "cus_123",
+          subscription_items: items
+        )
+
+        assert_requested :get, "#{Stripe.api_base}/v1/invoices/upcoming",
+          query: {
+            customer: "cus_123",
+            :'subscription_items[][plan]' => 'gold',
+            :'subscription_items[][quantity]' =>  2
+          }
```

Comment:
I got the follow error when I did ` 'subscription_items[][plan]': 'gold',`
```
/home/travis/build/stripe/stripe-ruby/test/stripe/invoice_test.rb:90: syntax error, unexpected ':', expecting => (SyntaxError)
            'subscription_items[][plan]': 'gold',
                                         ^
```
so I'm using `:'subscription_items[][plan]' => 'gold',` instead